packages/datagateway-download/src/__mocks__/axios.ts

```diff
@@ -10,19 +13,19 @@ const requests = {
       return Promise.resolve({
         data: {
           facilityName: 'LILS',
```

Comment:
Could this be `settings.facilityName` ?

Reply:
I didn't remove all instances of the hardcoded LILS as I didn't want to break anything but it's probably a good idea if the settings file has already been read in above